test/junit/scala/collection/immutable/LazyListLazinessTest.scala

```diff
@@ -889,6 +889,24 @@ class LazyListLazinessTest {
     assertLazyAll(op)
     assertRepeatedlyLazy(op)
   }
+
+  @Test
+  def sorted_properlyLazy(): Unit = {
+    val sorted = lazyListOp { ll => ll.sorted }
+    val sortBy = lazyListOp { ll => ll.sortBy(identity) }
+    val sortWith = lazyListOp { ll => ll.sortWith(_ < _) }
+
+
+    assertLazyAll(sorted)
+    assertLazyAll(sortBy)
+    assertLazyAll(sortWith)
+//    assertRepeatedlyLazy(sorted)
+//    assertRepeatedlyLazy(sortBy)
+//    assertRepeatedlyLazy(sortWith)
+//    genericLazyOp_properlyLazy(_.sorted)
```

Comment:
@NthPortal these commented tests all fail, not sure if that indicates a bug in LazyList or SeqView.Sorted or what.

Reply:
it indicates that I wrote all the tests and never documented what the test methods mean ;)

`assertRepeatedlyLazy` asserts that evaluating an element does not evaluate subsequent elements, which is not the case for sorting. and `genericLazyOp_properlyLazy` is just an aggregation of several methods, one of them being `assertRepeatedlyLazy`.

feel free to open a bug ticket to add docs to the test methods and assign it to me

Reply:
once you delete these commented-out lines, you can inline all the ops (without calling `lazyListOp`, which is just an identity method for driving type inference)